lib/internal/fs.js

```diff
@@ -18,16 +19,16 @@ const {
 
 function assertEncoding(encoding) {
   if (encoding && !Buffer.isEncoding(encoding)) {
-    throw new Error(`Unknown encoding: ${encoding}`);
+    throw new errors.TypeError('ERR_INVALID_OPT_VALUE', 'encoding', encoding);
```

Comment:
> Friendly reminder: We went from `ERR_UNKNOWN_ENCODING` to `ERR_INVALID_OPT_VALUE` to `ERR_INVALID_OPT_VALUE_ENCODING`.

I'm sorry to have made you run around... The new `Error` concept still has some undecided cases, you are just one of the first to tackle them...